cmd/sync/main.go

```diff
@@ -150,7 +154,7 @@ func main() {
 				}
 
 				if len(added) > 0 || len(removed) > 0 || len(updated) > 0 {
-					log.Printf("Updated Stream servers of %v; Added: %+v, Removed: %+v, Updated: %+v", upstream, added, removed, updated)
+					log.Printf("Updated Stream servers of %v for group %v ; Added: %+v, Removed: %+v, Updated: %+v", upstream.Name, upstream.ScalingGroup, getStreamUpstreamServerAddress(added), getStreamUpstreamServerAddress(removed), getStreamUpstreamServerAddress(updated))
```

Comment:
I recommend making the same change as with HTTP to reduce line length.

(something like:)
```suggestion
					addedAddresses := getStreamUpstreamServerAddress(added)
					removedAddresses := getStreamUpstreamServerAddress(removed)
					updatedAddresses := getStreamUpstreamServerAddress(updated)
					log.Printf("Updated Stream servers of %v for group %v ; Added: %+v, Removed: %+v, Updated: %+v",
						upstream.Name, upstream.ScalingGroup, addedAddresses, removedAddresses, updatedAddresses
```